### ISSUE TITLE
The bottom link was truncated by ‘OK’ button on ‘Migrate NuGet format to PackageReference’ dialog

### ISSUE THREAD

**Issue by v-nancyj** [NuGet/Home#7150]
## Details about Problem

NuGet version: NuGet Client release-4.8.0-RTM-4.8.0.5362
VS Version: D15.8 27924.00
OS Version: Windows-10-Enetrprise-RS3-RU-RU

## Detailed repro steps 
1.	Launch VS and click menu Tools->NuGet Package Manager->Package Manager Settings.
2.	Check ‘Allow format selection on first package install’ format.
3.	Create a new C# Console App (.NET Framework) via VS and click ‘Manage NuGet Packages…’
4.	Select a package in ‘Browse’ tab and click Install button.
5.	Select ‘Packages.config’ format and click OK button.
6.	Right-Click the “References” node and select the “Migrate packages.config to PackageReference…”
7.	Check the bottom link on ‘Migrate NuGet format to PackageReference’ dialog.

## Expected
The bottom link ‘Help me migrate to NuGet PackageReference’ showed entirely.	

### Actual
The bottom link was truncated by ‘OK’ button as below screenshot.
![bug](https://user-images.githubusercontent.com/20632775/43310815-2e014a9c-9178-11e8-8c65-ce7bb790bbde.png)

### Note
1. The issue only repro on RUS.
2. The issue repro on D15.8 27924.00 (Default NuGet Client Release-4.8.0-RTM\4.8.0.5348).

**Comment by zhili1208** [comment]
@rrelyea does this should be fixed in 15.8?

**Comment by v-luzh** [comment]
It is fixed on VS Main\35221.199 + NuGet Client Dev\6.12.0.81. I will close this bug soon.

**Comment by v-luzh** [comment]
It still reproes on VS D17.11\35103.136 with implicit NuGet Client Release-6.11.x\6.11.0.119.

**Comment by v-luzh** [comment]
It still reproes on VS Main\12206.68+NuGet Client Dev\7.12.0.4. So, I will reopen it.
